<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProjectSettings</class>
 <widget class="QWidget" name="ProjectSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ProjectSettings</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="MediaControl" name="pushButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>34</width>
         <height>34</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>34</width>
         <height>34</height>
        </size>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">background: white;</string>
       </property>
       <property name="icon">
        <iconset resource="icons.qrc">
         <normaloff>:/dialogs/project</normaloff>:/dialogs/project</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_14">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Ignored" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>QChromosome4D Document</string>
       </property>
       <property name="class" stdset="0">
        <string>heading</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <widget class="QWidget" name="tab_5">
      <attribute name="title">
       <string>Project</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <item row="0" column="0">
        <widget class="QScrollArea" name="scrollArea_2">
         <property name="focusPolicy">
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="scrollAreaWidgetContents_2">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>362</width>
            <height>214</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>Project Settings</string>
             </property>
             <property name="class" stdset="0">
              <string>section</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QFormLayout" name="formLayout_2">
             <property name="labelAlignment">
              <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
             </property>
             <property name="formAlignment">
              <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
             </property>
             <property name="horizontalSpacing">
              <number>6</number>
             </property>
             <property name="verticalSpacing">
              <number>6</number>
             </property>
             <item row="0" column="0">
              <widget class="Label" name="label_7">
               <property name="text">
                <string>FPS</string>
               </property>
               <property name="class" stdset="0">
                <string>item</string>
               </property>
              </widget>
             </item>
             <item row="2" column="0">
              <widget class="Label" name="label_8">
               <property name="text">
                <string>Minimum Time</string>
               </property>
               <property name="class" stdset="0">
                <string>item</string>
               </property>
              </widget>
             </item>
             <item row="4" column="0">
              <widget class="Label" name="label_9">
               <property name="text">
                <string>Preview Min. Time</string>
               </property>
               <property name="class" stdset="0">
                <string>item</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="SpinBox" name="spinBox">
               <property name="accelerated">
                <bool>true</bool>
               </property>
               <property name="minimum">
                <number>1</number>
               </property>
               <property name="maximum">
                <number>500</number>
               </property>
               <property name="value">
                <number>10</number>
               </property>
              </widget>
             </item>
             <item row="2" column="1">
              <widget class="SpinBox" name="spinBox_3">
               <property name="accelerated">
                <bool>true</bool>
               </property>
               <property name="suffix">
                <string>F</string>
               </property>
               <property name="maximum">
                <number>0</number>
               </property>
              </widget>
             </item>
             <item row="4" column="1">
              <widget class="SpinBox" name="spinBox_4">
               <property name="accelerated">
                <bool>true</bool>
               </property>
               <property name="suffix">
                <string>F</string>
               </property>
               <property name="maximum">
                <number>0</number>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="Label" name="label_10">
               <property name="text">
                <string>Document Time</string>
               </property>
               <property name="class" stdset="0">
                <string>item</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="SpinBox" name="spinBox_5">
               <property name="accelerated">
                <bool>true</bool>
               </property>
               <property name="suffix">
                <string>F</string>
               </property>
               <property name="maximum">
                <number>0</number>
               </property>
              </widget>
             </item>
             <item row="3" column="0">
              <widget class="Label" name="label_11">
               <property name="text">
                <string>Maximum Time</string>
               </property>
               <property name="class" stdset="0">
                <string>item</string>
               </property>
              </widget>
             </item>
             <item row="3" column="1">
              <widget class="SpinBox" name="spinBox_6">
               <property name="accelerated">
                <bool>true</bool>
               </property>
               <property name="suffix">
                <string>F</string>
               </property>
               <property name="maximum">
                <number>0</number>
               </property>
              </widget>
             </item>
             <item row="5" column="0">
              <widget class="Label" name="label_12">
               <property name="text">
                <string>Preview Max. Time</string>
               </property>
               <property name="class" stdset="0">
                <string>item</string>
               </property>
              </widget>
             </item>
             <item row="5" column="1">
              <widget class="SpinBox" name="spinBox_7">
               <property name="accelerated">
                <bool>true</bool>
               </property>
               <property name="suffix">
                <string>F</string>
               </property>
               <property name="maximum">
                <number>0</number>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <spacer name="verticalSpacer_2">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_4">
      <attribute name="title">
       <string>Info</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <item row="0" column="0">
        <widget class="QScrollArea" name="scrollArea_3">
         <property name="focusPolicy">
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="scrollAreaWidgetContents_3">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>362</width>
            <height>228</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <widget class="QLabel" name="label_13">
             <property name="text">
              <string>Info</string>
             </property>
             <property name="class" stdset="0">
              <string>section</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QFormLayout" name="formLayout_4">
             <property name="labelAlignment">
              <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
             </property>
             <property name="formAlignment">
              <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
             </property>
             <property name="horizontalSpacing">
              <number>6</number>
             </property>
             <property name="verticalSpacing">
              <number>6</number>
             </property>
             <item row="0" column="0">
              <widget class="Label" name="label_15">
               <property name="text">
                <string>Author</string>
               </property>
               <property name="class" stdset="0">
                <string>item</string>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="Label" name="label_16">
               <property name="text">
                <string>Info</string>
               </property>
               <property name="class" stdset="0">
                <string>item</string>
               </property>
              </widget>
             </item>
             <item row="2" column="0">
              <widget class="Label" name="label_17">
               <property name="text">
                <string>File Format</string>
               </property>
               <property name="class" stdset="0">
                <string>item</string>
               </property>
              </widget>
             </item>
             <item row="3" column="0">
              <widget class="Label" name="label_18">
               <property name="text">
                <string>File Version</string>
               </property>
               <property name="class" stdset="0">
                <string>item</string>
               </property>
              </widget>
             </item>
             <item row="4" column="0">
              <widget class="Label" name="label_19">
               <property name="text">
                <string>File Path</string>
               </property>
               <property name="class" stdset="0">
                <string>item</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="LineEdit" name="lineEdit_3"/>
             </item>
             <item row="2" column="1">
              <widget class="LineEdit" name="lineEdit_4">
               <property name="readOnly">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item row="3" column="1">
              <widget class="LineEdit" name="lineEdit_5">
               <property name="readOnly">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item row="4" column="1">
              <widget class="FileEdit" name="lineEdit_6">
               <property name="readOnly">
                <bool>true</bool>
               </property>
               <property name="type" stdset="0">
                <number>2</number>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="TextEdit" name="textEdit"/>
             </item>
            </layout>
           </item>
           <item>
            <spacer name="verticalSpacer_3">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>37</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MediaControl</class>
   <extends>QPushButton</extends>
   <header>mediacontrol.h</header>
  </customwidget>
  <customwidget>
   <class>FileEdit</class>
   <extends>QLineEdit</extends>
   <header>fileedit.h</header>
  </customwidget>
  <customwidget>
   <class>Label</class>
   <extends>QLabel</extends>
   <header>label.h</header>
  </customwidget>
  <customwidget>
   <class>SpinBox</class>
   <extends>QSpinBox</extends>
   <header>spinbox.h</header>
  </customwidget>
  <customwidget>
   <class>LineEdit</class>
   <extends>QLineEdit</extends>
   <header>lineedit.h</header>
  </customwidget>
  <customwidget>
   <class>TextEdit</class>
   <extends>QTextEdit</extends>
   <header>textedit.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>spinBox</tabstop>
  <tabstop>spinBox_5</tabstop>
  <tabstop>spinBox_3</tabstop>
  <tabstop>spinBox_6</tabstop>
  <tabstop>spinBox_4</tabstop>
  <tabstop>spinBox_7</tabstop>
  <tabstop>lineEdit_3</tabstop>
  <tabstop>textEdit</tabstop>
  <tabstop>lineEdit_4</tabstop>
  <tabstop>lineEdit_5</tabstop>
  <tabstop>lineEdit_6</tabstop>
 </tabstops>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
